src/jit/hwintrinsicxarch.cpp

```diff
@@ -547,9 +427,26 @@ GenTree* Compiler::impUnsupportedHWIntrinsic(unsigned              helper,
     }
 }
 
+//------------------------------------------------------------------------
+// impIsTableDrivenHWIntrinsic:
+//
+// Arguments:
+//    category - category of a HW intrinsic
+//
+// Return Value:
+//    returns true if this category can be table-driven in the importer
+//
+static bool impIsTableDrivenHWIntrinsic(HWIntrinsicCategory category, HWIntrinsicFlag flag)
+{
+    // TODO - make more categories to the table-driven framework
+    const bool tableDrivenCategory = category == HW_Category_SimpleSIMD || category == HW_Category_MemoryLoad ||
```

Comment:
Why can't this be as simple as `does not have flag 'NoCodeGen'`?

Reply:
Yes, here can just be `!Special` and `!NoCodeGen`. I will change and add comments.